test/controllers/RepresentativeDanImportVATControllerSpec.scala

```diff
@@ -257,4 +257,36 @@ class RepresentativeDanImportVATControllerSpec extends ControllerSpecBase {
     }
   }
 
+  "sameAccountNumber" when {
+
+    "same value for duty account number and vat account number submitted" in new Test {
+      override val userAnswers: Option[UserAnswers] = Some(
+        UserAnswers("some-cred-id")
+          .set(DefermentAccountPage, "1234567").success.value
+          .set(DefermentTypePage, "C").success.value
+          .set(AdditionalDefermentNumberPage, "1234567").success.value
+          .set(AdditionalDefermentTypePage, "A").success.value
+
+      )
+      controller.sameAccountNumber("1234567", "1234567", userAnswers.get) mustBe true
+      val request = fakeRequest.withFormUrlEncodedBody(buildForm(accountNumber = Some("1234567"), danType = Some("C")): _*)
+      controller.onSubmit(request)
```

Comment:
this tests seems to submit a request but doesn't verify anything about it (status code or errors etc.)